rust/src/ftp/constant.rs

```diff
@@ -0,0 +1,61 @@
+// FTP state progress values
+pub const FTP_STATE_NONE: u8 = 0;
```

Comment:
Why not use rust enum ?

The commit says `ftp: Convert enumerations to Rust` !

Reply:
With` repo(u8)` you should not need `typedef uint8_t FtpRequestCommand;` as it gets into rust-bindings.h from cbindgen